Fix debconf noninteractive setting

sercxanto opened the pull request:
When given on the kernel boot command line the setting has no effect,
debconf-get-selections still reports the standard setting "Dialog" after
initial login. This causes e.g. mysql-server to hang during installation,
because it interactively asks for a mysql root password (although mysql-server
is not part of the default installation, I adapted base.sh locally).

With the setting in presseed.cfg the installation finishes sucessfully.

Change-UUID: 9bf4aeeb20d14d8a9b28fda33906a613

sercxanto committed:
Fix debconf noninteractive setting

When given on the kernel boot command line the setting has no effect,
debconf-get-selections still reports the standard setting "Dialog" after
initial login. This causes e.g. mysql-server to hang during installation,
because it interactively asks for a mysql root password (although mysql-server
is not part of the default installation, I adapted base.sh locally).

With the setting in presseed.cfg the installation finishes sucessfully.

Change-UUID: 9bf4aeeb20d14d8a9b28fda33906a613

hnakamur commented:
@sercxanto Thanks for your pull request.
Could you also add `" debconf/frontend=noninteractive<wait>",` to boot_command in `virtualbox-iso`?

sercxanto commented:
Well I removed it in the boot command deliberately, because it didn't work and placed in in the preseed.cfg instead. The unattended mysql-server installation went fine then.

But now I realized now that I forgot to remove it in the vmware-iso section. Do you meant that I should remove it too in the vmware-iso section?

Do you have any evidence that "debconf/frontend=noninteractive<wait>" is working? I tried several variations including upper/lower case, but everytime the installatio hung, because it stuck in mysql-server configuration.

hnakamur commented:
No, I have no evidence that "debconf/frontend=noninteractive" is working.
I just thought the same changes should be applied to virtualbox-iso as well as vmware-iso.
If you forgot to remove them, please remove them from vmware-iso boot_command.

sercxanto committed:
Also remove debconf from cmd line of vmware

Forgot it in the previous change.

Change-UUID: 08c96960f6ae4f4da2f5c6ce7f3c4a40

hnakamur commented:
@sercxanto Thanks for your update. I merged your pull request now.
